tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/_indexstore.sh

```diff
@@ -35,6 +35,7 @@ readonly local_developer_dir="$DEVELOPER_DIR"
 
 
 $BAZEL_INSTALLERS_DIR/index-import \
+    -parallel-stride $(sysctl -n hw.physicalcpu)
```

Comment:
We might need a different code path for M1 VS intel here - becuase M1 has the efficeny cores. We can probably just take this number and divide it by 2

Reply:
Makes sense, will push the changes